README.md

```diff
@@ -25,28 +82,218 @@ This repository contains extension code for SQL Server input and output bindings
+
+    ```sql
+    INSERT INTO [dbo].[Employees] values (1, 'Hello', 'World', 'Microsoft', 'Functions')
+    ```
+
+- Congratulations! You have successfully created an Azure SQL Database! Make sure you complete [Quick Start](#Quick-Start) before continuing to the rest of the tutorial.
+
+### Input Binding Tutorial ###
+
+Note: This tutorial requires that the Azure SQL database is setup as shown in [Create Azure SQL Database](#Create-Azure-SQL-Database).
+
+- Open your app that you created in 'Set Up Your Local Environment' in VSCode
+- Press 'F1' and search for 'Azure Functions: Create Function'
+- Choose HttpTrigger -> (Provide a function name) -> Company.namespace -> anonymous
+- In the file that opens, replace the 'public static async Task< IActionResult > Run' block with the below code.
+
+    ```csharp
+    public static async Task<IActionResult> Run(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = "employees/{id}")] HttpRequest req,
+        ILogger log,
+        [Sql("select * from Employees where EmployeeId = @EmployeeId",
+        CommandType = System.Data.CommandType.Text,
+        Parameters = "@EmployeeId = {id}",
+        ConnectionStringSetting = "SqlConnectionString")]
+        IEnumerable<Employee> employee)
+    {
+        return new OkObjectResult(employee);
+    }
+    ```
+
+    *In the above, "select from Employees where EmployeeId = @EmployeeId" is the SQL script run by the input binding. The CommandType on the line below specifies whether the first line is a query or a stored procedure. The line below that determines the parameters. On the next line, the ConnectionStringSetting specifies that the app setting that contains the SQL connection string used to connect to the database is "SqlConnectionString." For more information on this, see the [Input Binding Samples](#Input-Binding-Samples) section*
+
+- Add 'using System.Collections.Generic;' to the namespaces list at the top of the page.
+- Currently, there is an error for the Ienumerable. We'll fix this by creating an Employee class.
+- Create a new file and call it 'Employee.cs'
+- Paste the below in the file. These are the column values of our SQL Database table.
+
+    ```csharp
+    namespace Company.Function {
+        public class Employee{
+            public int EmployeeId { get; set; }
+            public string LastName { get; set; }
+            public string FirstName { get; set; }
+            public string Company { get; set; }
+            public string Team { get; set; }
+        }
+    }
+    ```
+
+- Navigate back to your HttpTrigger file. We can ignore the 'Run' warning for now.
+- Open the local.settings.json file, and in the brackets for "Values," verify there is a 'SqlConnectionString.' If not, add it.
+- Hit 'F5' to run your code. Both an HttpTrigger and SQLBinding will run.
```

Comment:
I would instead say something like, "This will start up the Functions Host with a local HTTP Trigger and SQL Input Binding"

Reply:
This is perfect. I remember thinking about how to word that better earlier. Thanks Glenna